src/Domain/FunctionsOfTime/PiecewisePolynomial.hpp

```diff
@@ -108,9 +109,11 @@ class PiecewisePolynomial : public FunctionOfTime {
   // the values of that deriv order for all components.
   using value_type = std::array<DataVector, MaxDeriv + 1>;
 
-  std::vector<FunctionOfTimeHelpers::StoredInfo<MaxDeriv + 1>>
+  std::deque<FunctionOfTimeHelpers::StoredInfo<MaxDeriv + 1>>
       deriv_info_at_update_times_;
   double expiration_time_{std::numeric_limits<double>::lowest()};
+  alignas(64) std::atomic_uint64_t deriv_info_size_{};
+  char unused_padding_[64 - sizeof(std::atomic_uint64_t)];
```

Comment:
you need to make sure the size of `unused_padding_` is positive (otherwise you will get a lot of padding 😛 )

Reply:
Yea, I think `sizeof(std::atomic_uint64_t)` -> `sizeof(std::atomic_uint64_t) % 64` does it